config/environments/production.rb

```diff
@@ -22,7 +22,7 @@
 
   # Disable serving static files from the `/public` folder by default since
   # Apache or NGINX already handles this.
-  config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
+  # config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
```

Comment:
This line shouldn't be commented. The option remains disabled unless one has the `RAILS_SERVE_STATIC_FILES` env var defined, and that **is** something useful for local testing of the production environment. I personally use it and it should make it so y'all don't need an HTTP server running for serving static files.

We should use this approach to allow skipping the force_ssl options below for y'all to more easily test the production environment on your workstations without having to edit this config file. If this env var isn't present and production environment is used, it will still enable force_ssl like we want on the servers.

Please augment the below line as follows:
```ruby
config.force_ssl = ENV['RAILS_PROD_NOSSL'].blank?
```

Then y'all can start the Rails app with this command so it doesn't try to force HTTPS when you test production environment:

```bash
RAILS_ENV=production RAILS_PROD_NOSSL=1 SECRET_KEY_BASE=0 RAILS_SERVE_STATIC_FILES=1 rails s
```

Reply:
done